app/assets/v2/js/pages/join_tribe.js

```diff
@@ -83,3 +83,59 @@ const tribeLeader = () => {
 };
 
 tribeLeader();
+
+const newManageTribe = () => {
```

Comment:
this file is loaded in many pleases probably is better to move it to his own file or even loaded on that page only.

Reply:
could we omit this at this moment? i'll move this in another PR due i'll be working this weeks on tribes functionality.